framework/SupervisedLearning/KerasClassifier.py

```diff
@@ -32,21 +32,24 @@
 from sklearn import preprocessing
 import os
 
+import contrib.lazy.lazy_loader as lazy_loader
```

Comment:
josh we created in utils an lazyImporterUtils.py....maybe you want to modify it

Reply:
Thanks. (Also, is there a good method of testing the loading time of Raven?)

Reply:
"generally" I test it when the "PRA" plugin gets loaded.

Reply:
I know it is not ideal ;/

Reply:
Looks like I need to move the availOptimizers and other things to the ```__init__```

Reply:
I added -v to the debug python command in raven_framework and that can be used to check which modules are being loaded.

Reply:
top

Reply:
did you push the modified script?

Reply:
I did not push the modified raven_framework script.

Reply:
ah okay...maybe we can write it in the developer wiki ? just for reference?